src/index.cpp

```diff
@@ -4972,7 +4972,7 @@ static void writeIndex(OutputList &ol)
     ol.generateDoc(defFileName,defLine,Doxygen::mainPage.get(),nullptr,
                 Doxygen::mainPage->documentation(),true,false,
                 QCString(),false,false,
-                Config_getBool(MARKDOWN_SUPPORT),false);
+                Config_getBool(MARKDOWN_SUPPORT));
```

Comment:
Here also the Config_getBool(MARKDOWN_SUPPORT) parameter can be removed as it is already the default.